README.md

```diff
@@ -56,9 +56,11 @@ Execute `make test` from your bash compatible shell.  This will output the test
 ## Examples
 
 ### Trace
-You can use the [examples/AlwaysSampleTraceExample.php](/examples/AlwaysOnTraceExample.php) file to test out the reference implementation we have.  This example perfoms a sample trace with a grouping of 5 spans and POSTs the result to a local zipkin instance.
+You can use the [examples/AlwaysOnTraceExample.php](/examples/AlwaysOnTraceExample.php) file to test out the reference implementation we have for zipkin.  This example perfoms a sample trace with a grouping of 5 spans and POSTs the result to a local zipkin instance.
```

Comment:
Should we be renaming this file to `AlwaysOnZipkinTraceExample`?

Reply:
Thanks for the pointers, updating this PR now

Reply:
Updated @bobstrecansky